guidelines/releases/release-strategy.md

```diff
@@ -0,0 +1,173 @@
+
+At this time, the Release Manager publicly communicates the planned release schedule for Kyma. This communication should include key features or fixes expected to fall into the scope of the release. Possible communication channels include blog posts, social media, and the Core SIG meetings.
+
+### Development start
+After completing the planning, the Head of Capabilities explicitly hands over the release to the engineering teams during the handover meeting. At this point, the development phase of the new release officially starts.
+
+The Kyma developers add new release features to the `master` branch of the Kyma repositories.
+
+During the development phase, the Release Manager keeps track of the development with the help of the Scrum of Scrums (SoS) meetings held twice a week. This allows the Release Manager to take early actions required to keep the release on track. When closer to the release date, the Release Manager, the Head of Capabilities, and the Lead Product Engineer discuss the status of the release and decide if there are any additional checkpoints or actions to address.
+
+The Core SIG leaders include the external communication regarding the progress and status of each release to the agenda of the Core SIG bi-weekly meetings. The Release Manager, or a person appointed as a stand-in, presents the release status during those meetings.
+
+### Development end
+This is the last day of the period planned for the release development.
+
+### Release decision
+The Release Manager and the Product Leadership decide to publish the release, based on the status and readiness of the developed functionality. The Release Manager communicates this decision before the release execution starts.
+
+### Release execution
+The Release Engineer executes the release at any time of producing the release artifact. This could result in a release candidate or a final release that is not an automated nightly or weekly build. The release execution begins when the process starts and finishes after publishing all artifacts and the related documentation.
+
+During the execution of a release, Kyma developers run all end-to-end test scenarios. If any single scenario fails, the Release Engineer cannot publish the release.
+
+The Release Manager or Kyma developers notify the Kyma community when the release candidate is available. They pass this information to the public on Slack, requesting the community to validate and provide their short-term feedback. The deadline for the feedback is the day before the planned final release. Ideally, the final release is the same as the last release candidate, avoiding any additional changes. If the Kyma developers identify any release-related issues, they asses them and decide together with the Release Manager if they must fix them and produce a new release candidate.
```

Comment:
First line:
> Through the SIG Core, the Kyma community will be notified
Instead of "If the Kyma developers identify any release-related issues..."
> If any release-related issue is reported, it will be assessed to determine whether it would constitute a blocker for the release, in which case the Release Manager will be involved in the decision how to proceed. This will also be communicated to the SIG Core.

Reply:
I will strive to make it as "active" as possible and remove any passive voice from it (we try to avoid it in docs), preserving the meaning ;)

Reply:
...although I don't get the Core SIg involvement here?

Reply:
I rephrased that a bit - pls verify if that version is fine :)